### ISSUE TITLE
Unsync issue 

### ISSUE THREAD

**Issue by DodododoBA** [springlobby/springlobby#899]
Hi

I experience unsync issue when being in battleroom in the lobby for below engine & game version. Below are details:

Engine: 104.0.1-886-g0b800f1 maintenance
BA: ba-v10.15.sdz

Log:  [infolog.txt](https://github.com/springlobby/springlobby/files/2581171/infolog.txt)

Looks like I'm having issue with gadgets, 103 version works file tho.

Thanks
Dodo

**Comment by DodododoBA** [comment]
Singleplayer works:
![screen00001](https://user-images.githubusercontent.com/45040708/48490064-dbdd9a00-e824-11e8-8764-27f7c8f03321.jpg)

**Comment by abma** [comment]
> I experience unsync issue when being in battleroom

on which autohost?

which map?

**Comment by DodododoBA** [comment]
Hi

I checked 4 servers and I experience the same issue on every server. I can spec the game but cannot become a player due to unsync.

I tested different approaches with deploying the game, like: clearing all files, downloading it via lobby, installing it manually, keeping both versions and nothing worked. I ensured the correct engine is selected in the lobby's preferences. All that I could do from user perspective was done.

103 version works without problems.

Hosts of servers I checked: BAStable1, DarkEnergy1, [Ace]Pirine, DarkEnergy2

Thanks
